lms/djangoapps/edraak_university/models.py

```diff
@@ -59,3 +58,16 @@ def cleanup_id(id):
 
     class Meta:
         unique_together = ('user', 'course_key',)
+
+# Class related to university ID settings for a course
+class UniversityIDSetting(models.Model):
+    course_key = CourseKeyField(max_length=255, db_index=True)
+    registration_end_date = models.DateTimeField(default=None, null=True, blank=True)
+    terms_and_conditions_ar = models.TextField(null=True, blank=True)
+    terms_and_conditions_en = models.TextField(null=True, blank=True)
+
+    def get_terms_and_conditions(self):
+        if get_language() == 'ar':
```

Comment:
- [ ] move it to [helpers.py](https://github.com/Edraak/edx-platform/blob/569d9ef97b642a4ba0ddfc46b122b1d1557a3265/lms/djangoapps/edraak_university/helpers.py) under `get_translated_field()`